index.html

```diff
@@ -11122,7 +11122,7 @@ <h2>Type Maps</h2>
       <a>terms</a>,
       or the <a>keyword</a> <code>@none</code>,
       and the values MUST be <a>node objects</a>
-      or <a>strings</a> which expand to node obects.</p>
+      or <a>strings</a> which expand to <a>node objects</a>.</p>
```

Comment:
typo apart, I added this line because type maps may have plain IRIs as values.
Not entirely sure about the phrasing though.